lib/src/Uncertainty/Model/RandomVectorImplementation.cxx

```diff
@@ -184,6 +196,13 @@ Bool RandomVectorImplementation::isEvent() const
   return false;
 }
 
+RandomVector RandomVectorImplementation::getComposedEvent() const
+{
+  if (!isEvent())
+    throw InvalidArgumentException(HERE) << "Not an event.";
```

Comment:
I'm wondering if getComposedEvent should throw when there is no threshold/comparison operator
because subset/crossentropy algos need that and it's not checked (in master too :)
```
ev = ot.DomainEvent(ot.RandomVector(ot.Normal()), ot.Interval([-1.0], [1.0]))
ot.SubsetSampling(ev).run()
```
perhaps it could be a new property
or getComposedEvent should enforce that (and only be used from subset etc, as other simulation algo do not need the distance to the threshold and can keep using the simple form), (and be renamed asThresholdEvent ?)

anyway this is good enough for now